docs/sources/get-started/community_components.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@clayton-cornell I thought it's worth checking with you: do you think this is the right place for this doc?

Reply:
I think it should be OK here. the `get-started` section is slowly being adapted and cleaned up and should explain what Alloy is in more detail... including a description of components in general. This new content fits in here since it just describes what they are... and doesn't provide any instructions or reference content.